lib/segment/src/vector_storage/quantized/quantized_scorer_builder.rs

```diff
@@ -130,6 +178,41 @@ impl<'a> QuantizedScorerBuilder<'a> {
         }
     }
 
+    fn build_with_metric_bytes<TElement, TMetric>(
+        self,
+    ) -> OperationResult<Box<dyn QueryScorerBytes + 'a>>
+    where
+        TElement: PrimitiveVectorElement,
+        TMetric: Metric<TElement> + 'a,
+    {
+        match self.quantized_storage {
+            QuantizedVectorStorage::ScalarRam(storage) => {
+                self.new_quantized_scorer2::<TElement, TMetric>(storage)
+            }
+            QuantizedVectorStorage::ScalarMmap(storage) => {
+                self.new_quantized_scorer2::<TElement, TMetric>(storage)
+            }
+            QuantizedVectorStorage::PQRam(storage) => {
+                self.new_quantized_scorer2::<TElement, TMetric>(storage)
+            }
+            QuantizedVectorStorage::PQMmap(storage) => {
+                self.new_quantized_scorer2::<TElement, TMetric>(storage)
+            }
+            QuantizedVectorStorage::BinaryRam(storage) => {
+                self.new_quantized_scorer2::<TElement, TMetric>(storage)
+            }
+            QuantizedVectorStorage::BinaryMmap(storage) => {
+                self.new_quantized_scorer2::<TElement, TMetric>(storage)
+            }
+            QuantizedVectorStorage::ScalarRamMulti(_)
```

Comment:
Why do we skip multivector support? Is it temporary? In case of multivectors, even if we need to make additional disk reads, a new upcoming feature may be more efficient than search in mmap vector storage using regular hnsw